src/org/parosproxy/paros/view/View.java

```diff
@@ -168,6 +175,8 @@
     // ZAP: splash screen
     private SplashScreen splashScreen = null;
 
+    private static Map<AddOn.Status, StatusUI> statusMap = new HashMap<AddOn.Status, StatusUI>();

```

Comment:
Consider adding the map as an instance variable instead, it would "force" the use of the getSingleton() method which ensures that the map is initialised with the StatusUIs.